.wordlist.txt

```diff
@@ -209,6 +209,7 @@ klick
 Kodi
 Koel
 Komga
+kubeconfig
```

Comment:
Does it make sense to instead wrap that word into code tag `kubeconfig` (which are excluded from spell checking)? Or we could name it "client config" in the text. Generally I try to avoid adding too much commands, paths, variables, config keys/values and such into the word list and instead wrap them into code tags. It depends a bit on whether this is a well-known word when dealing with Kubernetes or not.

But minor thing, not a blocker.

Reply:
> It depends a bit on whether this is a well-known word when dealing with Kubernetes or not.

The kubeconfig is a fundamental part of how you interact with a kubernetes cluster, because it details how you connect and authenticate. So it is a quite well-known word. On the other hand, it is also a very specific thing, so wrapping it in code tags is fine. I'll do that and remove it from the wordlist.